v2/internal/pkg/delete/delete_images.go

```diff
@@ -62,24 +60,7 @@ func (o DeleteImages) WriteDeleteMetaData(images []v2alpha1.CopyImageSchema) err
 		if err != nil {
 			o.Log.Error("%v ", err)
 		}
-		i, err := o.Blobs.GatherBlobs(context.Background(), img.Source)
-		if err != nil {
-			o.Log.Error("%v image : %s", err, i)
-		}
-		if err != nil {
-			o.Log.Error(deleteImagesErrMsg, err)
-		}
-		var blobs []string
-		for k := range i {
-			blobs = append(blobs, k)
-			if err != nil {
-				o.Log.Error("unable to write blob %s %v", k, err)
-			}
-		}
-		sort.SliceStable(blobs, func(i, j int) bool {
-			return blobs[i] < blobs[j]
-		})
-		item.RelatedBlobs = blobs
+
 		items_map[img.Destination] = item
```

Comment:
@aguidirh nit: could you rename the variable to meet the go requirements "[Rename this local variable to match the regular expression " ^(_|[a-zA-Z0-9]+)$".